src/app/core/components/game-statistics/game-statistics.component.ts

```diff
@@ -13,4 +13,20 @@ import { Statistics } from '../../models/statistics.interface';
 })
 export class GameStatisticsComponent {
   @Input() statistics?: Statistics;
+
+  get playtimeInHours() {
+    if (!this.statistics) {
+      return 0;
+    }
+    return Math.round(this.statistics.time / 60);
+  }
+
+  get hoursPerEntry() {
+    if (!this.statistics || this.playtimeInHours === 0) {
```

Comment:
`playtimeInHours` of 0 is perfectly fine the the calculation further down. I would not exit because of that. feels more special than it it. the statistics blocker is fine though

Reply:
<img width="301" alt="Screenshot 2023-10-12 at 15 50 06" src="https://github.com/m-m-mic/Me-My-Shelf-and-I/assets/85286401/391ab2d5-44c7-478d-921d-a68598d6573d">

I have to catch it or else the 0 doesn't show up if the user has no items in their collection.